src/gui/screen_print_preview.hpp

```diff
@@ -4,13 +4,13 @@
 #include "window_header.hpp"
 #include "ff.h"
 
-#define PADDING          10
-#define SCREEN_WIDTH     240 //FIXME should be in display.h
-#define SCREEN_HEIGHT    320 //FIXME should be in display.h
-#define THUMBNAIL_HEIGHT 124
-#define TITLE_HEIGHT     24
-#define LINE_HEIGHT      15
-#define LINE_SPACING     5
+static const constexpr uint8_t PADDING = 10;
+static const constexpr uint8_t SCREEN_WIDTH = 240;   //FIXME should be in display.h
```

Comment:
SCREEN_WIDTH should have same type as SCREEN_HEIGHT 
I would prefer size_t, or just use GuiDefaults